apps/system/js/modal_dialog.js

```diff
@@ -131,6 +132,12 @@ var ModalDialog = {
         }
         break;
 
+      case 'system-resize':
+        if (this.isVisible()) {
```

Comment:
since this does the same thing as `keyboardchange`, you can just group it with that case and let it fall through